src/shared/ui/providers/auth-provider/auth-provider.tsx

```diff
@@ -0,0 +1,5 @@
+'use client';
+import { SessionProvider } from 'next-auth/react';
+import { ReactNode } from 'react';
+
+export const AuthProvider = ({ children }: { children: ReactNode }) => <SessionProvider>{children}</SessionProvider>;
```

Comment:
Do we really need to re-wrap session provider into auth provider? We don't add any additional logic, so consider using just session provider